src/main/java/frc/robot/Pivot/PivotIOSim.java

```diff
@@ -9,21 +9,25 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.system.plant.DCMotor;
 import edu.wpi.first.math.trajectory.TrapezoidProfile;
-import edu.wpi.first.math.util.Units;
 import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
 
-public class ArmIOSim implements ArmIO {
+public class PivotIOSim implements PivotIO {
+  private final SingleJointedArmSim pivotSim;
+
   // TODO: change to actual values
-  private final SingleJointedArmSim armSim =
-      new SingleJointedArmSim(
-          DCMotor.getKrakenX60Foc(1),
-          ArmSubsystem.PIVOT_RATIO,
-          0.1,
-          Units.inchesToMeters(25), // arm length
-          ArmSubsystem.MIN_ANGLE.getRadians(), // min angle
-          ArmSubsystem.MAX_ANGLE.getRadians(), // max angle
-          true,
-          0.0);
+  public PivotIOSim(
+      double PIVOT_RATIO, double MIN_ANGLE_RADIANS, double MAX_ANGLE_RADIANS, double length) {
```

Comment:
```suggestion
      double pivotRatio, double minAngleRadians, double maxAngleRadians, double length) {
```
parameters should be `camelCase`